t/e2e/jail/jail_rpc_test.go

```diff
@@ -102,7 +102,7 @@ func (s *JailRPCTestSuite) TestRegressionGetTransactionReceipt() {
 
 	// note: transaction hash is assumed to be invalid
 	got := rpcClient.CallRaw(`{"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["0xbbebf28d0a3a3cbb38e6053a5b21f08f82c62b0c145a17b1c4313cac3f68ae7c"],"id":7}`)
-	expected := `{"jsonrpc":"2.0","id":7,"result":null}`
+	expected := `{"jsonrpc":"2.0","id":7,"error":{"code":-32000,"message":"unknown transaction"}}`
```

Comment:
There is no counterpart of `eth_getTransactionReceipt`?

Reply:
Moved this commit to #702, since this is actually part of the migration to 1.8.1. In any case, it looks that due to changes in 1.8.1, there are no situations where a `nil` `result` will be returned (an `error` will always be returned in those situations), so we can remove this code: https://github.com/status-im/status-go/blob/develop/vendor/github.com/ethereum/go-ethereum/ethclient/ethclient.go#L250

Reply:
Sorry, I read `unknown method` instead of `unknown transaction`. So, we are still good and `eth_getTransactionReceipt` method is available but the output for invalid transaction hashes changed.

Reply:
Exactly. And from what I could gather from our codebase, we're not looking specifically for the resulting error message, so it seems there are no other changes required.